Use check- and radio buttons in Allocate disk space dialogs

jpnurmi opened the pull request:
To get rid of the highlight background which is meant for list views
and looks a bit awkward on a frameless background. Our custom desktop
oriented check and radio buttons were added for exactly this reason.

jpnurmi commented:
| Before | After |
|---|---|
| ![Screenshot from 2021-09-15 09-37-44](https://user-images.githubusercontent.com/140617/133391938-bb53770a-8d8f-463d-a2df-35f3b2b0ccf5.png) | ![Screenshot from 2021-09-15 09-37-32](https://user-images.githubusercontent.com/140617/133391951-ed497839-e883-493c-8dc2-6a7a227cfebb.png) |

oSoMoN commented on packages/ubuntu_desktop_installer/lib/pages/allocate_disk_space/allocate_disk_space_dialogs.dart:
```diff
@@ -191,7 +191,7 @@ class _RadioValueTile<T> extends StatelessWidget {
       valueListenable: groupValue,
       builder: (context, groupValue, child) {
-        return RadioListTile<T>(
+        return RadioButton<T>(
```
Vertical alignment of the two columns is broken, as evidenced by your "after" screenshot (and tested locally).

jpnurmi replied:
Good catch, thanks!

jpnurmi commented:
I rebased and added a separate commit to "Share and expose kRadioSize constant" in ubuntu_wizard. I hope it's ok to include it here as a separate commit, if you could rebase or merge to keep it so, to avoid creating a dependency PR.

jpnurmi commented:
![Kuvakaappaus - 2021-09-21 10-28-56](https://user-images.githubusercontent.com/140617/134138930-a19eb2b2-176f-4f81-bbdd-0f7c92ea8498.png)
![Kuvakaappaus - 2021-09-21 10-28-44](https://user-images.githubusercontent.com/140617/134138941-9944c1a9-7acd-4262-840f-07a5f13f341b.png)

oSoMoN commented:
LGTM now, thanks!
